.github/workflows/build.yml

```diff
@@ -0,0 +1,120 @@
+name: Build
+
+on: [push, pull_request]
```

Comment:
I was wondering why there is no build success indicator in the PR, it seems this is due to the forking repo model GitHub uses. It seems that the workflows only fires in the base repo. They have a workaround to set `pull_request_target` instead of `pull_request`. [Some details here.](https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/)

If I were to guess, when the PR is merged the automation will be triggered on the main repo. But at that point if there is a build error it is a little late for that information.

Reply:
Ok thanks, we can test this on the next PR that is put in. If it does not work then please put in a PR to fix it.